assets/src/styles/blocks/Accordion/AccordionEditorStyle.scss

```diff
@@ -1,5 +1,5 @@
-@import "../../master-theme/assets/src/scss/base/variables";
 @import "../../master-theme/assets/src/scss/base/colors";
+@import "../../master-theme/assets/src/scss/base/variables";
```

Comment:
It's not ideal to have to make sure that we always import the `colors` file first, we could probably import it directly in the `variables` file but since that would require more changes and testing, I think we can keep it this way for now and refactor later!